### DISCUSSION TITLE
Queries on TimeBase Views

### DISCUSSION THREAD

**Discussion by ankitsoin** [finos/TimeBase-CE#292] [Q&A]
Hi @alex-karpovich,

Pleas help us with few queries on Timebase Views:

1. Do we have spaces inside views?
2. Is it possible to load single space of stream data into the existing/new view?
3. I have a view which is created with feature Live = OFF. Can you please help us knowing how to refresh the view manually once we load the new data in base stream?

**Comment by alex-karpovich** [comment]
Hi @ankitsoin 
1. For now we do not support `spaces` for the views, assuming that `views` is not used for 1:1 transformation.
2. use filters for your query, if you have specific distribution (by symbols or by date)
3. I suggest to have LIVE = ON in this case, otherwise you need to re-create view.
